### ISSUE TITLE
Assigning a number to a citizen = Maximum liberty

### ISSUE THREAD

**Issue by lunokhod** [liberland/constitution#188]
I would like to argue that this phrase of the constitution reduces liberty:

    §II.17. No law shall allow the Public Administration to assign a number to any Individual

Let's just take as an example the case of voting. The government will have a database of citizens, and this database should have enough information to allow the citizen to prove that they are in fact a citizen and allowed to vote. It should contain enough information that if the citizen lost their passport, that they could replace it. For "normal" governments, this database would include your name, address, date of birth, height, weight, a photo, and maybe more. Given that this would be stored in an electronic database, it would be trivial to assign a unique number to each person, but this number would not be very useful in of itself: It would only allow to distinguish between two people with the same name (there at at least two people with public web pages who have my exact name..).

The potential loss of liberty arises because of identity theft: It would not be too hard for a person to convince authorities that they are one of the citizens in the government database. This would allow them to be issued a new passport, voiding the passport of the real citizen. If this person decided to vote, it would deprive the real citizen of the liberty of voting.

So, how do we rectify this problem? The solution is to assign a unique number to each person: The novelty is that this would not be a useless ID number that most governments use for database purposes, but rather it would correspond to the public PGP key of a cryptographic public/private key pair, and the number would be provided by the citizen. The public key is simply a long number, and it allows the citizen to prove that they are in fact who they claim. To do this, the authorities would simply ask the person to sign a message with their key, and the public authority would then verify the signature using the public key that is in their database.

In addition to guaranteeing all liberties provided for by the constitution, assigning a public key (a number) to a citizen also provides for increased privacy. If the citizen decided to submit a PGP key, and allowed the government to use this for identification purposes, the citizen entry in the database would no longer need to contain other information for identification purposes, such as biometric data, and knowledge of certain information like date of birth (which can often be found elsewhere anyways...). Regardless, given that a person might loose their PGP private key, it might be prudent to have a minimum of biometric data in the database (such as a photo and height) for backup purposes.

Thus, I propose to either 
1. remove this phrase (the rest of the section is fine), or
2. require that the assigning of any identifying data to the individual (PGP keys, biometric data, address, etc) be done only with the consent of the individual.

Given that I can not see how a Liberland citizen could exist with no identifying information in a database, I prefer option 1.

**Comment by terrorist96** [comment]
The PA **can** assign a number to an individual, if that person consents (see the rest of the section).

**Comment by lunokhod** [comment]
So they can assign a number, but no law will allow the PA to assign a number to an individual? This appears to be logically inconsistent.

In my opinion, removing this phrase does not weaken the rest of the section.

**Comment by KacperZajc** [comment]
It is a long sentence but it clearly states that a number can be assigned with consent. Naturally, Citizens who want to take part in elections and referendums will register.

**Comment by lunokhod** [comment]
Actually, the section does not state that they can assign a number with consent. It only states that data should be kept confidential, used only for the purposes for which they were collected, and **shared** only with consent.

Let me ask the question differently:

What is the purpose of the phrase "No law shall allow the Public Administration to assign a number to any Individual" when the rest of the section implies implicitly that the government can keep records on citizens, and that these records can only be used for the purpose they were collected and shared only with consent? Perhaps I am missing something, but the phrase I am complaining about doesn't seem to provide any form of additional benefit to the citizen.

**Comment by KacperZajc** [comment]
You clearly misread this section.

The point is that the government (Public Administration) cannot assign a number to an individual without consent because only prisoners are numbers, free people are not. Nor can anyone form the government of the law enforcement demand that a law-abiding person identifies themselves. This protects privacy, you are left alone, that is your right.

The second part guarantees that any data collected with consent will not be used for purposes other than it was collected for thereby consented to.

**Comment by lunokhod** [comment]
Even after re-re-re-reading the section, and even when considering external information concerning the intent of the authors, I still find the first 14 words to be at best very confusing. I would suggest clarifying this by replacing it with something along these lines:

> No law shall allow the Public Administration to collect, assign, or store any information about an individual without the consent of the Individual.

In addition to clarification, this resolves a second logical problem which is perhaps more damning: The original wording of this section allows the PA to collect information about an Individual without their consent as long as this data is not shared, and as long as this information is used only for the purposes for which it was collected!

**Comment by KacperZajc** [comment]
The fact that you cant find the meaning we want to express, tells us something about the wording of this provision. I will consider restating.

**Comment by terrorist96** [comment]
Here's an easy fix. 

> §II.17. No law shall allow the Public Administration to assign a number to any Individual, nor shall he or she be required to procure, carry, or otherwise maintain identification materials of any form**,** without one's express and informed consent

Commas are important. :smile:

**Comment by KacperZajc** [comment]
Here is what I think:

§II.17. No law shall allow the Public Administration to assign a number to any Individual without one's express and informed consent; nor shall he or she be required to procure, carry, or otherwise maintain identification materials of any form.

§II.XX No law shall allow for any Individual to be compelled to identify themselves to any Agent of the Public Administration, otherwise upon arrest or pursuant to a Warrant; no Individual shall be searched; medically examined or otherwise tested unless pursuant to a Warrant.

**Comment by terrorist96** [comment]
II.7 change he or she to 'one' 

XX change the semicolon to a comma after 'searched'

**Comment by KacperZajc** [comment]
§II.17. No law shall allow the Public Administration to assign a number to any Individual without one's express and informed consent; nor shall one be required to procure, carry, or otherwise maintain identification materials of any form, or share their personal information with any branch of the Public Administration unless pursuant to a Warrant.

§II.XX No law shall allow for any Individual to be compelled to identify themselves to any Agent of the Public Administration, otherwise upon arrest or pursuant to a Warrant; no Individual shall be searched, medically examined or otherwise tested unless pursuant to a Warrant.

**Comment by FellowTraveler** [comment]
I like this wording. Correct me if I'm wrong:

1. The government cannot force you to be assigned a number, but you can voluntarily choose to be assigned a number, for example, for the purposes of being a registered voter.

2. However, in this example (voter registration) the voter ID number could not be used by any other branch of the PA, or even viewed without a warrant.

If those are true, that's fine with me.

There are several purposes behind this banning of number assignment. One is to quell any religious fears about a "number of the beast" being imposed on the people to buy or sell. A more important purpose is because identification numbers are a tool of oppression as history shows us. We saw the culmination of this oppression when victims in the Nazi camps were forced to submit to a tattoo of their identification number. _"I am not a number, I am a free man."_

Those who live in the United States have seen the slippery slope with the social security number. When the SSN was first imposed, the government promised the population that it would _"never be used as a form of identification."_ In fact, the SSN cards used to explicitly promise this on the back of the card. That promise no longer appears on the cards, and now, decades later, the SSN is used all over the place as a form of identification. It's actually the primary form of identification for a person in the United States, and often used for identity theft.

The promise was broken, it's a slippery slope, and the best way to prevent it is to ban it entirely. After all, the government can still do its job: there's no reason why the cops can't still arrest and prosecute thieves and murderers, even without assigning identification numbers to the populace.

**Comment by KacperZajc** [comment]
correct

**Comment by lunokhod** [comment]
As someone already noted in another thread: If your name is stored in an electronic database, you are by definition associated with a number (that is, the binary, hex, or decimal equivalent). The problem with that number (i.e., your name) is that it is not necessarily unique; there could be two people with the same name. Now, let us presume that all citizens are listed in a public register. Would it be a problem is there were two identical entries? (Let us presume for the sake of argument that both individual's refused to allow other identifying information, such as date of birth, to be recorded.) Perhaps you don't consider that a problem, but I do: Those two entries might correspond to the same person, which could lead to them voting twice.

Now let us presume that there is only one person listed in the citizen registry: Their name is unique and can be used in the exact same way as an identification number could be. If you want to quell any number phobia, you would have to remove the person's name from the citizen registry all together, which would in effect revoke their citizenship (as they would have no way to prove it; the current certificate of honorary citizenship is easily forged).

tl;dr *If your name is unique in Liberland, you have already been assigned a unique identifying number.*

**Comment by terrorist96** [comment]
I don't see why there should be a registry to begin with full of unconsenting people's names.

My view is that citizenship will require a unique number (think passport). And choosing to become a citizen is a free choice. There's nothing preventing someone from just being a resident in Liberland (except for Croatia, currently).

**Comment by lunokhod** [comment]
So, if I understand, 

1. Being a citizen is a choice.
2. Every citizen is assigned a number. If they don't consent to having the number assigned, they can just be a resident instead of a citizen.

Therefore, every citizen has a number assigned to them, contrary to the prevision in the draft constitution that is noted in the original post.

**Comment by terrorist96** [comment]
It wouldn't be contrary since they would have the person's consent.

**Comment by lunokhod** [comment]
Let me state your point differently:

If you are a citizen of Liberland, you will have a number assigned to you. If you don't want to consent to this, don't apply to be a citizen.

**Comment by terrorist96** [comment]
Correct.
